examples/platform/silabs/display/lcd.cpp

```diff
@@ -25,6 +25,8 @@
 #include "dmd.h"
 #include "glib.h"
 
+#include "rsi_chip.h"
```

Comment:
This should be enclosed in same macro since EFR32 uses this file as well

Reply:
done